dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/event/RequestEvent.java

```diff
@@ -58,10 +59,18 @@ public static RequestEvent toRequestEvent(ApplicationModel applicationModel, Str
                                               MetricsDispatcher metricsDispatcher, DefaultMetricsCollector collector,
                                               Invocation invocation, String side) {
         MethodMetric methodMetric = new MethodMetric(applicationModel, invocation);
+        RequestEvent requestEvent = getRequestEvent(applicationModel, appName, metricsDispatcher, collector, invocation,
+            side, methodMetric, MetricsSupport.getInterfaceName(invocation));
+        return requestEvent;
+    }
+
+
+
+    private static RequestEvent getRequestEvent(ApplicationModel applicationModel, String appName, MetricsDispatcher metricsDispatcher, DefaultMetricsCollector collector, Invocation invocation, String side, ServiceKeyMetric methodMetric, String serviceKey) {
```

Comment:
This method of extraction seems unnecessary, there are too many parameters, and it is only used once.